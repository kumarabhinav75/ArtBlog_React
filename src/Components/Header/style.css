header {
  width: 100%;
  background: black;
  padding-top: 15px;
}

.heading {
  font-size: 1.5em;
  margin: 0;
}

.heading-content {
  margin : 0px;
  font-size: 1.2em;
}
.container {
  width:75%;
  margin: auto;
}

.nav-list {
  color: white;
  list-style-type: none;
  display: flex;
  justify-content: flex-end;
  margin: 0px;
}

.nav-items {
  width: 40%;
  margin-bottom: 5px;
  display: flex;
  justify-content: space-between;
}