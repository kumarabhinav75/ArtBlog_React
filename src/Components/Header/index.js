import React from 'react';
import './style.css';

const Header = () => {
  return (
    <header>
      <div className="container">
        <p className="heading">The Artifact</p>
        <p className="heading-content"><i>Culture & Art blog</i></p>
        <ul className="nav-list">
          <div className="nav-items">
            <li>Blog</li>
            <li>About</li>
            <li>Contact</li>
          </div>
        </ul>
      </div>

    </header>
  );
}

export default Header;