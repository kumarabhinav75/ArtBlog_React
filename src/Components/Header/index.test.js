import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './index';

describe('Header', () => {
  it ('should match the snapshot', () => {
    const tree = renderer.create(<Header />);
    expect(tree).toMatchSnapshot();
  });
})