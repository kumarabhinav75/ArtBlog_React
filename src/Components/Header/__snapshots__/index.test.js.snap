// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<header>
  <div
    className="container"
  >
    <p
      className="heading"
    >
      The Artifact
    </p>
    <p
      className="heading-content"
    >
      <i>
        Culture & Art blog
      </i>
    </p>
    <ul
      className="nav-list"
    >
      <div
        className="nav-items"
      >
        <li>
          Blog
        </li>
        <li>
          About
        </li>
        <li>
          Contact
        </li>
      </div>
    </ul>
  </div>
</header>
`;
